content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -382,6 +382,33 @@ The nodes are where your workloads (containers and pods, etc) run. To add new no
 kubeadm join --token <token> <master-ip>:<master-port> --discovery-token-ca-cert-hash sha256:<hash>
 ```
 
+If you do not have the token, you can get it by running the following command on the master node:
+
+``` bash
+# kubeadm token list
```

Comment:
See https://kubernetes.io/docs/home/contribute/style-guide/#code-snippet-formatting - don't include the command prompt like "#", also further below